include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
It is definitively a packaging issue, but it isn't a trivial issue. Autotools actually doesn't recommend installing the generated config.h in the /usr/local/include (which is where the library is installed). It comes with a lot of #defines that, unless renaming, could do some serious harm. This is why I left it out when working on autotools and left it for future work.
(see e.g. http://stackoverflow.com/questions/19586211/correct-installation-of-config-h-for-shared-library-using-autotools)

So, I try to let as little of CppUTest headers be dependent on config. That tends to lead to nicer headers anyways as avoiding #ifdef is almsot always a good idea. So, in this case...